packages/@ember/-internals/glimmer/tests/integration/components/component-template-test.js

```diff
@@ -31,8 +25,6 @@ moduleFor(
     [`${testUnless(
       DEPRECATIONS.DEPRECATE_COMPONENT_TEMPLATE_RESOLVING.isRemoved
     )} it takes precedence over resolver`]() {
```

Comment:
If this test doesn't trigger the deprecation, why would it be removed when the deprecation is removed?

Reply:
because it's a scenario we don't need to care about.

Currently, we're testing if `setComponentTemplate` is used and no resolution happens.

But resolution can _never_ happen once the deprecated is removed, so the whole test becomes invalid.